### ISSUE TITLE
Another error introduced while create production artifact

### ISSUE THREAD

**Issue by mappuji** [ole-vi/planet#197]
Another error while build prod version.

Reported in https://www.travis-ci.org/ole-vi/planet/builds/305904116#L3065

```
10% building modules 5/8 modules 3 active .../bootstrap/dist/css/bootstrap.min.cssNode#moveTo was deprecated. Use Container#append.
Date: 2017-11-22T17:05:45.732Z
Hash: aebd03020d5fbdf44327
Time: 17528ms
chunk {0} styles.2867e867484647fdb917.bundle.css (styles) 205 kB [initial] [rendered]
chunk {1} polyfills.3bc34265385d52184eab.bundle.js (polyfills) 86 bytes [initial] [rendered]
chunk {2} main.e402deade8b026b7d50e.bundle.js (main) 84 bytes [initial] [rendered]
chunk {3} inline.c72ec803e3bc84a3c0d1.bundle.js (inline) 1.45 kB [entry] [rendered]
ERROR in Error at /ng-app/src/app/shared/dialogs/dialogs-form.component.html(1,18): Property 'onSubmit' does not exist on type 'DialogsFormComponent'.
```

For this problem I'll propose to have git hook when create a commit to prevent problem
in `package.json` add
```
"config": {
    "ghooks": {
      "pre-commit": "ng build --prod",
    }
  },
"devDependencies": {
    "ghooks": "^1.3.2"
}
```

**Comment by mappuji** [comment]
@paulbert @leomaxi

**Comment by mappuji** [comment]
Is it fixed with https://github.com/ole-vi/planet/commit/4382fb82f8d40febaefc9592bef7352dbf181295 ?

Can I closed #227 and #197 ?

**Comment by paulbert** [comment]
Yes, that got it working in Travis so closing this issue.
